docs/list.md

```diff
@@ -424,6 +424,20 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 
 ### O
 
+#### **OpenBB**
+
+- **Swag**: T-shirt, Cap
+- **Requirements**:
+    - To get both T-shirt and Cap:
+        - Create 1 Custom OpenBB Extension.
+    - To get 1 of the both items, you need to do one of the following:
+        - 5 no-code contributions
+        - 3 code/performance enhancements
+        - 1 non-extension feature
+- **How to sign up**: No special signup required.
```

Comment:
Hi @vaibhav-009! The OpenBB team does have a swag signup process available here: https://openbb.co/blog/hacktoberfest-join-us-in-celebrating-open-source

Can you please summarize things and include the steps from that page where it says "Once you've completed your contribution, please follow these steps:"?

Reply:
Hey @crweiner I have made the requested changes. can you please review it.